src/registrar/tests/test_views.py

```diff
@@ -349,79 +349,6 @@ def test_home_doesnt_delete_other_domain_requests(self):
                         # clean up
                         domain_request.delete()
 
-    @less_console_noise_decorator
-    def test_home_deletes_domain_request_and_orphans(self):
```

Comment:
Remove test_home_deletes_domain_request_and_orphans since Contacts should be separate from DomainRequest with Creator data deriving from user profile instead of domain request